Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -164,6 +164,115 @@ public static bool AddLease(string clientId, string uuid, BrokerRole brokerRole,
             }
         }
 
+        public bool UpdateAcces(string uuid, bool updateAccessPass = false, bool updateAccessCode = false)
+        {
+            if (!updateAccessPass && !updateAccessCode)
+            {
+                return false;
+            }
+
+            using (var db = Mem.Database.Open())
+            {
+                OrderTable order = db.Single<OrderTable>(x => x.OrderId == uuid && !x.Deleted);
+
+                if (order != null)
+                {
+                    List<OrderItemsTable> updatedOrderItems = new List<OrderItemsTable>();
+                    List<OrderItemsTable> orderItems = db.Select<OrderItemsTable>(x => x.OrderId == order.OrderId);
+
+                    foreach (OrderItemsTable orderItem in orderItems)
+                    {
+                        if (orderItem.Status == BookingStatus.Confirmed || orderItem.Status == BookingStatus.Proposed || orderItem.Status == BookingStatus.None)
+                        {
+                            updatedOrderItems.Add(orderItem);
+
+                            if (updateAccessCode)
+                            {
+                                orderItem.PinCode = Faker.Random.String(length: 6, minChar: '0', maxChar: '9');
+                            }
+
+                            if (updateAccessPass)
+                            {
+                                orderItem.ImageUrl = Faker.Image.PlaceholderUrl(width: 25, height: 25);
+                                orderItem.BarCodeText = Faker.Random.String(length: 10, minChar: '0', maxChar: '9');
+                            }
+
+                            db.Save(orderItem);
```

Comment:
Note this also needs 
```
                    order.Modified = DateTimeOffset.Now.UtcTicks;
                    order.VisibleInFeed = true;
                    db.Update(order);
```

In the same way as UpdateOpportunityAttendance has this, as without updating `order.Modified` they won't come through the feed.